pkg/webhook/resourcequota/validation/validation.go

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Resource quotas are strictly enterprise feature, hence it would be nice to wrap it and use only for EE builds.

Reply:
aah missed that.
sure will do, thx :)